src/fleet/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '1.6.2'
+VERSION = '1.6.3'
```

Comment:
Only bump up this version number will trigger a release. So please remove this line change